src/components/resources/ResourcesOverview.js

```diff
@@ -1,12 +1,22 @@
 import styled from 'styled-components';
 import {Resource} from './Resource';
+import {nanoid} from '@reduxjs/toolkit';
+import {useSelector} from 'react-redux';
+
+export const ResourcesOverview = () => {
+  const currentUserResources = useSelector(state => state.currentUserResources);
+
+  let currentRess = [];
```

Comment:
Again, I would use a `state` here.

Reply:
i have changed it to a const. 

items from the stats are inserted there
this array is just a kind of placeholder here the assembled things are created